From bb0d191c842547b33e46b2ba22343da71147c206 Mon Sep 17 00:00:00 2001
From: Abhishek Joshi <abjoshi@broadcom.com>
Date: Sun, 17 Aug 2014 18:21:15 -0700
Subject: [PATCH 0581/1532] ucore: added ucore file for multi-domain dm apps

also added app-param node under dom2
in xlp9xx-config.dts
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp9xx-config.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/mips/boot/dts/xlp9xx-config.dts b/arch/mips/boot/dts/xlp9xx-config.dts
index 9282584..3e46f11 100644
--- a/arch/mips/boot/dts/xlp9xx-config.dts
+++ b/arch/mips/boot/dts/xlp9xx-config.dts
@@ -255,6 +255,9 @@
 				rsa = <0x0>;
 				fmn = <0x0>;
 			};
+			app-param {
+				onlinemask = <>;
+			};
 		};
 	};
 
-- 
1.9.1

